src/js/view/controls/controls.js

```diff
@@ -101,6 +101,7 @@ export default class Controls extends Events {
         this.logo = this.playerContainer.querySelector('.jw-logo');
 
         const touchMode = model.get('touchMode');
+        const floatingElement = model.get('isFloating') ? this.wrapperElement : this.playerContainer;
```

Comment:
Not sure about the naming for this const?

Reply:
lgtm!